drivers/net/dsa/lantiq_gsw_mdio.c

```diff
@@ -358,12 +417,19 @@ static void gsw_mdio_remove(struct mdio_device *pmdiodev)
 /* MaxLinear GSW120 & GSW125 */
 static const struct gsw_hw_info gsw_120 = {
 	/* TODO WARP-5828: Determine what these values should be */
-	.max_ports = 4,
-	.cpu_port = 1,
+	.max_ports = 6,
```

Comment:
Also will probably make this driver easily support the 6 port part.

Reply:
The supported interface info for each port has been added via a new `.ops` function in WARP-5926.
It currently assumes the same supported connection types for all MDIO-connected GSW parts (which is accurate for the GSW12x & GSW140).

Reply:
Not sure exactly what tact I will take regarding a mechanism to mandate that ports 2 & 3 are disabled for the GSW12x, but I do know it will be covered in a future PR, not this one 👍